pages/accountLists/[accountListId]/reports/partnerGivingAnalysis/[[...contactId]].page.test.tsx

```diff
@@ -41,9 +41,9 @@ const TestingComponent: React.FC<TestingComponentProps> = ({
   };
 
   const mocks = {
```

Comment:
When we define `mocks` as a separate variable instead of passing them directly to `GqlMockedProvider` we loose a little type safety. We can get it back by adding `satisfies DeepPartial<Mocks>` to the end of the definition.

```ts
const mocks = { /* ... */ } satisfies DeepPartial<Mocks>;
```